python/ray/train/gbdt_trainer.py

```diff
@@ -24,18 +24,37 @@ def _convert_scaling_config_to_ray_params(
     ray_params_cls: Type["xgboost_ray.RayParams"],
     default_ray_params: Optional[Dict[str, Any]] = None,
```

Comment:
Might be out of scope for this PR, but would be nice to add a docstring for this. Am a bit confused on scaling_config vs. default_ray_params.